code/modules/mob/mob.dm

```diff
@@ -418,7 +418,7 @@ var/next_mob_id = 0
 	if(mind)
 		mind.show_memory(src)
 	else
-		src << "The game appears to have misplaced your mind datum, so we can't show you your notes."
+		src << "You don't have a mind datum for some reason, so you can't look at your notes, if you had any."
```

Comment:
We probably SHOULD throw an error at this point, because something really went wrong.